### PR TITLE
Bump verison to 1.19.6-prerelease

### PR DESCRIPTION
**Pull request by lucaspimentel**

### CODE REVIEW

File: deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Files.Managed.Netcoreapp31.wxs [hunk 1]
```diff
@@ -30,6 +35,11 @@
               Source="$(var.TracerHomeDirectory)\netcoreapp3.1\Sigil.dll"
               KeyPath="yes" Checksum="yes"/>
       </Component>
+      <Component Win64="$(var.Win64)">
+        <File Id="netcoreapp31_lzma.dll"
+              Source="$(var.TracerHomeDirectory)\netcoreapp3.1\lzma.dll"
```
Comment (lucaspimentel) [inline deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Files.Managed.Netcoreapp31.wxs:40]: @tonyredondo: Do you know why this dependency got added? It is something we use for duck typing in specifically in .NET Core 3.1?

Comment (tonyredondo) [reply deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Files.Managed.Netcoreapp31.wxs:40]: No, that’s weird. Ducktyping only have two dependencies and for netstandard : https://github.com/DataDog/dd-trace-dotnet/blob/106a8e074944f2b2926a82554f0beeb39b69a924/src/Datadog.Trace.DuckTyping/Datadog.Trace.DuckTyping.csproj#L17

Comment (lucaspimentel) [reply deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Files.Managed.Netcoreapp31.wxs:40]: When I `publish` the project on my machine the output has `runtimes/win7-x64/native/lzma.dll`.

Comment (tonyredondo) [reply deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Files.Managed.Netcoreapp31.wxs:40]: I’ll check but as you can see there is not a direct reference, something is kicking in for netcoreapp target.

Comment (zacharycmontoya) [reply deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Files.Managed.Netcoreapp31.wxs:40]: @lucaspimentel we can ignore this dependency. This is not used by the product and is only used transitively (through Sigil I believe) for a dotnet-zip tool. Aka it's not used

Comment (lucaspimentel) [reply deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Files.Managed.Netcoreapp31.wxs]: Thanks. Removed it 080635dc.

File: src/Datadog.Trace.DuckTyping/Datadog.Trace.DuckTyping.csproj [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <!-- NuGet -->
-    <Version>1.19.3</Version>
+    <Version>1.19.6-prerelease</Version>
```
Comment (zacharycmontoya) [inline src/Datadog.Trace.DuckTyping/Datadog.Trace.DuckTyping.csproj:5]: You should revert this change, that way when this PR and PR #987 are both merged, only that one modifies the file

Comment (lucaspimentel) [reply src/Datadog.Trace.DuckTyping/Datadog.Trace.DuckTyping.csproj:5]: I don't follow. We still want the other version attributes to get bumped here even if the other PR keeps the assembly version at 1.0.0. If I revert this it will stay at `1.19.3` even with the other PR. Did I miss something?

Comment (lucaspimentel) [reply src/Datadog.Trace.DuckTyping/Datadog.Trace.DuckTyping.csproj:5]: i.e. we want both changes

Comment (zacharycmontoya) [reply src/Datadog.Trace.DuckTyping/Datadog.Trace.DuckTyping.csproj:5]: Ah nevermind, you're right we do want both changes. My bad 🙃

**Review by zacharycmontoya — APPROVED** [review]

**Review by tonyredondo — APPROVED** [review]
